### ISSUE TITLE
Provide way to get back to deck homepage

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#385]
In order for users to navigate back to the deck homepage, which will have useful guiding content, there should be a link in the side nav.

Acceptance criteria
- The first item in the nav should be "Overview" which should take the users to the home of the deck.

**Comment by brittag** [comment]
I'd simply title this "Guide" and put it at the top of the sidebar.

**Comment by thisisdano** [comment]
This seems like a good solution for now. In the future, the "deck home" might act more like an overview, but that's miles ahead.

**Comment by jeremiak** [comment]
Right now, clicking on the big cloud.gov logo will bring you back to the deck homepage. Not sure if that is sufficient, but wanted to flag

**Comment by brittag** [comment]
Ah yeah, as a user I assumed clicking that cloud.gov logo would take me to the cloud.gov homepage since that's the behavior for other parts of the cloud.gov site, so I didn't try clicking that when I wanted to get back to the Deck homepage. :)

**Comment by brittag** [comment]
"Overview" is totally fine to me as a title for this item too! That's vague enough that it can mean either "help/guide" or "summary of your orgs/spaces/apps". :)

**Comment by jeremiak** [comment]
Yup, agreed that it is weird that the logo acts differently in different contexts

**Comment by thisisdano** [comment]
I, too, agree that there is some dissonance when the logo has a different target in different contexts, but it's also a fairly common pattern for the site home to change based on your signin status. In fact, it would be difficult to think of a web app where the primary top-left nav (usually the logo or a variation) targeted the marketing home rather than the "dashboard home" once the user is signed in.

Counter-examples welcome, tho!

**Comment by jeremiak** [comment]
I tried out two things:

1. Simply added "Overview" as the first `<li>` in the sidenav
2. Using the `title_bar` classes that are present on [docs.cloud.gov](https://docs.cloud.gov)

Screenshot for 1:
![Screenshot of "Overview" added as the first `<li>` in the sidenav](https://s3.amazonaws.com/f.cl.ly/items/361V0J2B2f2X091I2f0v/Screen%20Shot%202016-06-21%20at%2010.52.37%20PM.png?v=a3412853)

Screenshot for 2:
![Screenshot of using the `title_bar` classes that are present on cloud.gov documentation site](https://s3.amazonaws.com/f.cl.ly/items/1j3t212o3p3M0M1a1N0q/Screen%20Shot%202016-06-21%20at%2010.58.56%20PM.png?v=e32872d1)

Both of these were under 5 minute attempts, so this wouldn't be the final implementation but there are already some problems with each approach.

In the first option, "Overview" seems to be out of place alongside all of the organizations. However, since it is currently just an empty view and the way to navigate through organizations with the sidebar it isn't the worst.

The second option is a bit wonkier. When we made the deck go full window width, we didn't update the `.title_bar` classes to reflect that. We're also not including Font Awesome anymore in the library, which should've resulted in a breaking change release but I think that was before we had our release process documented and tested.

I'm inclined to just go with the first option for now because its _really_ fast and easy, but I think we can/should alter the structure of the side navigation soon after we get this in the hands of a few people.

Thoughts @thisisdano @brittag and @msecret?
